js/navigation.js

```diff
@@ -0,0 +1,47 @@
+const pageHome = document.querySelector('[data-js=home]');
+const pageBookmark = document.querySelector('[data-js=bookmarked]');
+const pageCreate = document.querySelector('[data-js=create]');
+const pageProfile = document.querySelector('[data-js=profile]');
+
+const linkHome = document.querySelector('[data-js=link-home]');
+const linkBookmark = document.querySelector('[data-js=link-bookmark]');
+const linkCreate = document.querySelector('[data-js=link-create]');
+const linkProfile = document.querySelector('[data-js=link-profile]');
+
+const allPages = document.querySelectorAll('.page');
+
+const headline = document.querySelector('[data-js=headline]');
+
+export default function navigation() {
+
+linkHome.addEventListener('click', () => {
+  allPages.forEach((page) => {
+    page.classList.remove('current');
+  })
+  pageHome.classList.add('current');
```

Comment:
I thing here is the change to  headline.textContent = "Home" also neccessary. maybe you are coming back from another page?